admin/partials/smaily-for-wp-admin-form.php

```diff
@@ -1,13 +1,13 @@
 <?php
-$api_credentials = $this->api_credentials;
+$api_credentials_set = $this->api_credentials_set;
```

Comment:
Might as well just use `$this->api_credentials_set` directly.

Reply:
Yep, better to keep it consistent.
Renamed it to has_credentials with 5eaf1ac

Reply:
I meant you could directly use `$this->has_credentials` without assigning it to `$has_credentials` variable.

Reply:
Ah, ok, I see. d25413e